group/assignment6/ParkingLot/Car.java

```diff
@@ -2,34 +2,37 @@
 
 import java.util.Objects;
 
+/**
+ * Represents a car with an ID.

```

Comment:
Represents a car with a unique ID.

Reply:
This is not enforced within the class itself though so I don't think we should put this.